sys/include/net/gnrc/udp.h

```diff
@@ -54,6 +54,13 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Default UDP port range for dynamic source port allocation
+ */
+#define GNRC_UDP_PORTRANGE_MIN (49152U)
```

Comment:
mhm, I took the naming from `sysctl -a | grep portrange` on macOS:

```
net.inet.ip.portrange.hilast: 65535
net.inet.ip.portrange.hifirst: 49152
net.inet.ip.portrange.last: 65535
net.inet.ip.portrange.first: 49152
net.inet.ip.portrange.lowlast: 600
net.inet.ip.portrange.lowfirst: 1023
```

IMHO _EPHEMERAL_ isn't that commonly used?